src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageSearch/PackageSearchResultJsonRenderer.cs

```diff
@@ -16,32 +13,30 @@ internal class PackageSearchResultJsonRenderer : IPackageSearchResultRenderer
     {
         private ILoggerWithColor _logger;
         private PackageSearchVerbosity _verbosity;
+        private bool _exactMatch;
         private PackageSearchMainOutput _packageSearchMainOutput;
 
-        public PackageSearchResultJsonRenderer(ILoggerWithColor loggerWithColor, PackageSearchVerbosity verbosity)
+        public PackageSearchResultJsonRenderer(ILoggerWithColor loggerWithColor, PackageSearchVerbosity verbosity, bool exactMatch)
         {
             _logger = loggerWithColor;
             _verbosity = verbosity;
+            _exactMatch = exactMatch;
         }
         public async void Add(PackageSource source, IEnumerable<IPackageSearchMetadata> completedSearch)
         {
             PackageSearchResult packageSearchResult = new PackageSearchResult(source.Name);
 
             foreach (IPackageSearchMetadata metadata in completedSearch)
             {
-                if (_verbosity == PackageSearchVerbosity.Minimal)
-                {
-                    packageSearchResult.Packages.Add(new MinimalVerbosityPackage(metadata));
-                }
-                else if (_verbosity == PackageSearchVerbosity.Detailed)
+                string deprecation = "";
```

Comment:
looks like this is an unused variable. Is it needed here? If yes, can we `string.Empty` instead of  `""`?